yorkie/backend/sync/etcd/pubsub.go

```
@@ -41,11 +48,78 @@ func (c *Client) Publish(
 	publisherID *time.ActorID,
 	topic string,
 	event sync.DocEvent,
+) {
+	c.PublishToLocal(publisherID, topic, event)
+
+	c.memberMapMu.RLock()
+	defer c.memberMapMu.RUnlock()
+
+	for _, member := range c.Members() {
+		memberAddr := member.RPCAddr
+		if memberAddr != c.pubSub.AgentInfo.RPCAddr {
+			c.addClusterClient(memberAddr)
+		}
+
+		if memberAddr != c.pubSub.AgentInfo.RPCAddr {
+			c.publishToMember(
+				memberAddr,
+				publisherID,
+				topic,
+				event,
+			)
+		}
+	}
+}
+
+// PublishToLocal publishes the given event to the given Topic.
+func (c *Client) PublishToLocal(
+	publisherID *time.ActorID,
+	topic string,
+	event sync.DocEvent,
 ) {
 	c.pubSub.Publish(publisherID, topic, event)
+}
+
+// addClusterClient activates the cluster grpc server and creates a client.
+func (c *Client) addClusterClient(addr string) {
+	c.clusterClinetMapMu.Lock()
+	defer c.clusterClinetMapMu.Unlock()
+
+	if _, ok := c.clusterClientMap[addr]; !ok {
+		conn, err := grpc.Dial(addr, grpc.WithInsecure())
+		if err != nil {
+			log.Logger.Error(err)
```

Wouldn't it be better to return an error or try again?